diff --git a/pyproject.toml b/pyproject.toml
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -32,7 +32,7 @@
 ]

 [build-system] # How pip and other frontends should build this project
-requires = ["py-build-cmake~=0.2.0a12"]
+requires = ["py-build-cmake"]
 build-backend = "py_build_cmake.build"

 [tool.py-build-cmake.module] # Where to find the Python module to package
@@ -48,8 +48,6 @@
 build_args = ["-j"]
 options = {"LOTTIE_MODULE" = "OFF"}
 install_components = ["python_module"]
-python_abi = 'abi3'
-abi3_minimum_cpython_version = 37
 
 [tool.py-build-cmake.linux.cmake] # Linux-specific options
 config = "Release"
